content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -405,6 +410,39 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry environment
+
+In OpenTelemetry environments, you can use the [OpenTelemetry Collector][16] to collect and batch telemetry data (like metrics, traces, and logs) and the [OpenTelemetry Exporter][17] to send the data to Datadog (without the Datadog Agent).
+
+To setup unified service tagging in an OpenTelemetry environment:
+
+1. Set the following attributes in your OpenTelemetry configuration to ensure proper tagging:
+
+   ```yaml
+   service.name: "<SERVICE>"
+   deployment.environment: "<ENV>"
+   service.version: "<VERSION>"
+   ```
+   
+1. To [correlate OpenTelemetry language SDK logs and traces][18], translate the OpenTelemetry `TraceId` and `SpanId` properties into the Datadog format, and ensure your logs are sent as JSON files. Your language-level logs must become Datadog attributes for the trace-log correlation to work.
```

Comment:
> ensure your logs are sent as JSON files

It's unclear to me what you mean by this. There is an [OTLP/JSON format](https://opentelemetry.io/docs/specs/otlp/#json-protobuf-enco), but using this is not required for consuming logs: the OpenTelemetry Collector supports both.


> Your language-level logs must become Datadog attributes

I also don't understand this, what does that mean?